### PR TITLE
Testdesign

### PR DESCRIPTION
**Pull request by irisepinto**
Is it a good idea to change the title of the website to "Blueberries"?

### CODE REVIEW

**Comment by yenapaek** [comment]
> Is it a good idea to change the title of the website to "Blueberries"?

Honestly, I'm not sure!

**Comment by irisepinto** [comment]
> > Is it a good idea to change the title of the website to "Blueberries"?
> 
> Honestly, I'm not sure!

Let's keep the website name the same but commit the other changes to the master!

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <meta name="description" content="">
   <meta name="author" content="">
 
-  <title>New Website</title>
+  <title>Blueberries</title>
```
Comment (irisepinto) [inline index.html]: We are vetoing this!

File: index.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <a class="js-scroll-trigger" href="#page-top">Website Title</a>
       </li>
       <li class="sidebar-nav-item">
-        <a class="js-scroll-trigger" href="#page-top">Home</a>
+        <a class="js-scroll-trigger" href="#page-top">Landing Page</a>
```
Comment (irisepinto) [inline index.html]: Good idea!

**Pull request closed without merging**